Dockerfile-devel-curve

```diff
@@ -43,7 +43,7 @@ RUN yum install -y \
     if [ "$TARGETARCH" = "amd64" ]; then \
     OCI_ARCH=x86_64; \
     elif [ "$TARGETARCH" = "arm64" ]; then \
-    OCI_ARCH=aarch64; \
+    OCI_ARCH=arm64; \
```

Comment:
I suggest not to use `OCI_ARCH` here. OCI is the driver of Oracle database. You can change it to `BAZEL_ARCH` or something like that.

Reply:
> I suggest not to use `OCI_ARCH` here. OCI is the driver of Oracle database. You can change it to `BAZEL_ARCH` or something like that.

fix